content/news/2024-03-13-welcome-global-team/index.en.md

```diff
@@ -0,0 +1,84 @@
+author: "Leadership Team"
+type: blog
+date: "2024-03-13"
+output: html_document
+image: img/welcome.PNG
+tags: 
+- community
+categories: 
+- R-Ladies
+summary: R-Ladies Global Team has 15 new members!
+slug: /news/new-to-global-team-2023/
+---
+
+![Thumbnail photos of 15 new global team members on purple background.](img/welcome.PNG)
+
+<!---
+link to ppt that contains this image on rladies google account
+https://docs.google.com/presentation/d/1ZnCmaO_gBSKSbtx_1tTicQSxnTJZaYGc/edit#slide=id.p1
+-->
+
+# Overview
+
+In 2023 we [announced openings](https://rladies.org/news/2023-04-11-global-team-recruiting/) for roles on the R-Ladies Global Team that support organizational efforts and facilitate the growth of the R-Ladies community.
+We are delighted to welcome 15 new members to the R-Ladies Global Team.
+R-Ladies is 100% volunteer-driven, and we are very grateful to those who 
+support us with their time and effort to champion our [mission](https://rladies.org/about-us/mission/).
+
+
+
+# New members
+
+|                                     |                           |                    |
+|-------------------------------------|---------------------------|--------------------|
+| ![](img/glenda_mendieta.jpg)        | Glenda Mendieta           | Translation        |
+| ![](img/nicola_rennie.jpg)           | Nicola Rennie             | Campaigns          |
+| ![](img/sara_acevedo.jpg)           | Sara Acevedo              | Code of Conduct    |
+| ![](img/hebah_bukhari.jpg)          | Hebah Bukhari             | Community Slack    |
+| ![](img/priyanka_gagneja.jpg)       | Priyanka Gagneja          | Community Slack    |
+| ![](img/renata_hirota.png)          | Renata Hirota             | Blog               |
+| ![](img/virginia_garcia_alonso.jpg) | Virginia A. García Alonso | Mentoring          |
+| ![](img/nic_crane.jpg)          | Nic Crane                 | Meetup Pro         |
+| ![](img/alyssa_columbus.png)        | Alyssa Columbus           | Chapter Onboarding |
+| ![](img/luana_atunes.png)           | Luana Atunes              | Abstract Review    |
+| ![](img/sayantika_banik.png)        | Sayantika Banik           | Abstract Review    |
+| ![](img/cosima_meyer.png)           | Cosima Meyer              | Website            |
+| ![](img/andrea_gomez_vargas.png)    | Andrea Gómez Vargas       | Website            |
+| ![](img/leena_el_seed.jpg)          | Leena El Seed             | Conference Liaison |
+| ![](img/daniela_vazquez.png)        | Daniela Vázquez           | Social Media       |
+
+# Onboarding needs
+
+Between 2020-2023, membership in the R-Ladies global organization increased 
+while the availability of volunteers remained limited. At the conclusion of this
+time period several global team members retired. We are indebted for their 
+efforts that helped the organization thrive and grateful for their time spent 
+on the global team. 
+
+As the shift in personnel occurred, new teams and team structures were identified
```

Comment:
the word personnel made me think on people we pay for their services. I would change for some word that don't have that meaning.  I would use volunteers.

Reply:
great catch, thank you! updated!